ci_tools/bot_messages/welcome_friend.txt

```diff
@@ -1,5 +1,3 @@
 Hello again! Thank you for this new pull request :star_struck:.
 
-Don't forget to let me know when it is complete with the command `/bot mark as ready`.
-
-Here is your checklist:
+Here is your checklist. Please tick items of when you have completed them or determined that they are not necessary for this pull request:
```

Comment:
Typo: of -> off

Reply:
Fixed [0583484](https://github.com/pyccel/pyccel/pull/1449/commits/05834841d9ac63ca53ddfc17a44bc7ee453efbd2)